runtime/lua/vim/treesitter/_headings.lua

```diff
@@ -87,8 +87,10 @@ local get_headings = vim.func._memoize(hash_tick, function(bufnr)
   return headings
 end)
 
---- Show a table of contents for the help buffer in a loclist
-function M.show_toc()
+--- Shows an Outline (table of contents) of the current buffer, in the loclist.
+---
+--- @param title? string
+function M.show_toc(title)
```

Comment:
This should really be an `opts` table, since we may want to add a section filter later.

Reply:
it's an internal function so we can always change it later?